mounts/org.osbuild.xfs

```diff
@@ -26,16 +26,23 @@ SCHEMA_2 = """
   },
   "options": {
     "type": "object",
-    "additionalProperties": true
+    "properties": {
+      "readonly": {
+        "description": "mount the source as a readonly device",
+        "type": "boolean"
+      }
+    },
+    "additionalProperties": false
   }
 }
 """
 
 
 class XfsMount(mounts.FileSystemMountService):
 
-    def translate_options(self, _options: Dict):
-        return ["-t", "xfs"]
+    @staticmethod
+    def translate_options(options: Dict):
+        return ["-t", "xfs"] + mounts.FileSystemMountService.translate_options(options)
```

Comment:
If you keep the base as normal method rather than `staticmethod`, you can change this to `super().translate_options(options)`.